src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java

```diff
@@ -97,7 +96,7 @@ public void actionPerformed(final ActionEvent e) {
                     + "\nConvert to: "
                     + audioSelect.getConvertTo()
                     + "\nQuality: "
-                    + quality.getQualityAudio()
```

Comment:
Maybe these lines should be one line  + "\nConvert to: " + audioSelect.getConvertTo()